docs/src/changelog.md

```diff
@@ -14,6 +14,10 @@ If your currently installed *Quality-time* version is not the latest version, pl
 
 ## [Unreleased]
 
+### Added
+
+- Next to ignoring branches, tags, and jobs, also allow for including branches, tags, and jobs when measuring failed CI-jobs with GitLab as source. Closes [#4520](https://github.com/ICTU/quality-time/issues/4520).
```

Comment:
Does the implementation in `GitLabJobsBase` only apply this to `failed CI-jobs` ? The change in data model suggests more metrics :)

Reply:
Yes, you're right. I'll have to see if we need unit tests for those other metrics as well.

Reply:
Extra unit tests not needed, the four metrics share a base class that test the new parameters.